data/json/npcs/robofac/robofac_intercom/robofac_intercom_prototypes.json

```diff
@@ -543,7 +543,7 @@
     "id": "TALK_ROBOFAC_INTERCOM_MAG_BUY",
     "//": "Buying HWP Magazines",
     "type": "talk_topic",
-    "dynamic_line": "Certainly, we can immediately provide you with a loaded magazine of 5.56 NATO, 7.62 NATO, 9mm Parabellum, or 2 shotgun magazines for 1 coin.",
+    "dynamic_line": "Certainly, we can immediately provide you with a loaded magazine of 5.56 NATO, 7.62 NATO, 9mm Parabellum, or 2 shotgun magazines for 1 coin. We also offer drum magazines for 3 coins, since some of our operatives have found a sustained high-caliber firepower quite necessary lately.",
```

Comment:
```suggestion
    "dynamic_line": "Certainly, we can immediately provide you with a loaded magazine of 5.56 NATO, 7.62 NATO, 9mm Parabellum, or 2 shotgun magazines for 1 coin.  We also offer drum magazines for 3 coins, since some of our operatives have found a sustained high-caliber firepower quite necessary lately.",
```